invenio/values.yaml

```diff
@@ -28,6 +26,11 @@ invenio:
       level: "WARNING"
       celery: true
       environment: "qa"
+  mail:
+    suppress: true
```

Comment:
is this different from `enabled`? (which we may seem to use to decide if inject or not mail configurations in `invenio/templates/configurations/invenio.yaml`)

Reply:
Thank you for spotting this! The `enabled` was wrong. I have removed it! :)